index.html

```diff
@@ -57,7 +57,7 @@ <h3>I’m a software developer! </h3>
             <h1>My Recent Works</h1>
             <div class="projects">
                 <div class="multi_stories flex">
-                    <img src="/images/Img Placeholder.png" alt="Image Placeholder">
+                    <img src="/images/Img-Placeholder.png" alt="Image Placeholder">
                     <h3>Multi-Post Stories</h3>
                     <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
                     <ul class="flex">
```

Comment:
- [ ] As requested by the previous reviewer, I would suggest you work on the project cards as currently, I can only see one card instead of 6 cards. Kindly fix this as this will give more confidence in the ability to replicate any given design 👍 👍